external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventHubSpout.java

```diff
@@ -121,8 +121,8 @@ public void preparePartitions(Map config, int totalTasks, int taskIndex, SpoutOu
         zkEndpointAddress = sb.toString();
       }
       stateStore = new ZookeeperStateStore(zkEndpointAddress,
-          (Integer)config.get(Config.STORM_ZOOKEEPER_RETRY_TIMES),
-          (Integer)config.get(Config.STORM_ZOOKEEPER_RETRY_INTERVAL));
+          Integer.parseInt(config.get(Config.STORM_ZOOKEEPER_RETRY_TIMES).toString()),
```

Comment:
Sorry, misread this bit. Read it as parseInt(conf).toString instead of parseInt(conf.toString) :)